src/pages/deduplicate/index.tsx

```diff
@@ -0,0 +1,309 @@
+        }
+    };
+
+    return (
+        <GalleryContext.Provider
+            value={{
+                ...defaultGalleryContext,
+                closeMessageDialog,
+                syncWithRemote,
+                setDialogMessage,
+                startLoading,
+                finishLoading,
+                setNotificationAttributes,
+                setBlockingLoad,
+            }}>
+            {blockingLoad && (
+                <LoadingOverlay>
+                    <EnteSpinner />
+                </LoadingOverlay>
+            )}
+
+            <LoadingBar color="#51cd7c" ref={loadingBar} />
+            <AlertBanner bannerMessage={bannerMessage} />
+            <ToastNotification
+                attributes={notificationAttributes}
+                clearAttributes={clearNotificationAttributes}
+            />
+            <MessageDialog
+                size="lg"
+                show={messageDialogView}
+                onHide={closeMessageDialog}
+                attributes={dialogMessage}
+            />
+            {duplicateFiles.length > 0 ? (
+                <PhotoFrame
+                    files={duplicateFiles}
+                    setFiles={setDuplicateFiles}
+                    syncWithRemote={syncWithRemote}
+                    favItemIds={new Set()}
+                    setSelected={setSelected}
+                    selected={selected}
+                    isFirstLoad={isFirstLoad}
+                    isInSearchMode={false}
+                    deleted={[]}
+                    activeCollection={ALL_SECTION}
+                    isSharedCollection={false}
+                    enableDownload={true}
+                    deduplicating={{
+                        clubByTime: clubByTime,
+                        fileSizeMap: fileSizeMap,
+                    }}
+                />
+            ) : (
+                <b
+                    style={{
+                        fontSize: '2em',
+                        textAlign: 'center',
+                        marginTop: '20%',
+                    }}>
+                    {constants.NO_DUPLICATES_FOUND}
+                </b>
+            )}
+
+            {selected.count > 0 ? (
+                <SelectedDuplicatesOptions
+                    setDialogMessage={setDialogMessage}
+                    deleteFileHelper={deleteFileHelper}
+                    count={selected.count}
+                    clearSelection={clearSelection}
+                    clubByTime={clubByTime}
+                    setClubByTime={setClubByTime}
+                />
+            ) : (
+                <div
+                    style={{
+                        position: 'absolute',
+                        top: '1em',
+                        left: '1em',
+                        zIndex: 10,
+                    }}
+                    onClick={() => {
+                        router.push(PAGES.GALLERY);
+                    }}>
+                    <LeftArrow />
```

Comment:
let wrap the svg with [`IconButton`](https://github.com/ente-io/bada-frame/blob/86a692f908d9e0bc240203a78a7db78cf43ad534/src/components/Container.ts#L21),  it will improve its styling